### ISSUE TITLE
files/root/createhostkeytab.sh always completes successfully 

### ISSUE THREAD

**Issue by v1peractual** [ncsa/puppet-profile_system_auth#9]
files/root/createhostkeytab.sh always completes succuessfully due to 'rm' being the last command. The errors from the previous commands need to be trapped and exited correctly.

**Comment by v1peractual** [comment]
For reference:
```
RANDSTRING=`head -c 16 /dev/random  | base64 | grep -o . | sort -R | tr -d "\n" | head -c 14`
REQCLASS1=`date | base64 | tr -dc A-Z | grep -o . | sort -R | tr -d "\n" | head -c2`
REQCLASS2=`date | base64 | tr -dc a-z | grep -o . | sort -R | tr -d "\n" | head -c2`
REQCLASS3=`date | tr -dc 0-9 | grep -o . | sort -R | tr -d "\n" | head -c2`
REQCHARS=`echo $REQCLASS1$REQCLASS2$REQCLASS3`
TEMPPASS=`echo "$RANDSTRING$REQCHARS" | grep -o . | sort -R | tr -d "\n"`

echo "$1" | base64 --decode > /root/createhost.keytab

echo -e "$TEMPPASS\n$TEMPPASS" | kadmin -kt /root/createhost.keytab -p $2/createhost@NCSA.EDU -q "addprinc host/$(hostname -f)@NCSA.EDU"
echo -e "$TEMPPASS" | kadmin -p host/$(hostname -f)@NCSA.EDU -q "ktadd host/$(hostname -f)@NCSA.EDU"
rm -f /root/createhost.keytab
```

**Comment by billglick** [comment]
This was resolved in commit 5a81bd1c98786ba3f738fe443af918376c52d0b2 by adding the following to the top of the script:

```
set -e
```
